app/controllers/katello/api/v2/repository_sets_controller.rb

```diff
@@ -52,11 +52,21 @@ def available_repositories
         end
       end
 
+      sorted_groups = repos.sort_by do |repo|
```

Comment:
Naming nitpick! This variable has the word 'groups' in it. Personally, I would expect it to be a Hash structure of some kind with the keys representing the groups for example. There is no actual "grouping" here in terms of the data structure returned. What about `sorted_groups` ?